.htconfig.php-dist

```diff
@@ -75,4 +75,4 @@
   //At which health value should we start removing profiles?
   'remove_profile_health_threshold' => -60
   
-);
\ No newline at end of file
+);
```

Comment:
The "dist" file is called `htconfig.php` without the dot in Friendica. I think it should be the same here.